en/docs/get-started/streaming-quick-start-guide.md

```diff
@@ -1,142 +1,221 @@
-        @map(type = 'csv'))
-    define stream SweetProductionStream (name string, amount double);
-
-    @sink(type = 'kafka', bootstrap.servers = "localhost:9092", topic = "total_production", is.binary.message = "false", partition.no = "0",
-        @map(type = 'json'))
-    define stream TotalProductionStream (name string, amount double);
-
-    -- Simple Siddhi query to calculate production totals.
-    @info(name = 'query1')
-    from SweetProductionStream
-    select name, sum(amount) as amount
-    group by name
-    insert into TotalProductionStream;
+
+    @App:description('Receive events via an xml file located in a given directory and write sum of the amount to websocket server as a json event')
+
+     @source(type = 'file', mode = "TEXT.FULL", dir.uri = 'file:/Users/foo/files', enclosing.element = "/events/event", enable.streaming.xml.content = "true", 
+    	@attributes(name = "name", amount = "amount"),
+	    @map(type = 'xml'))
+     @sink(type = 'log')
+    define stream SweetsProductionStream (name string, amount double);
+
+    @sink(type = 'websocket-server', host = "localhost", port = "8025",
+	    @map(type = 'json'))
+    define stream ProductionAlertStream (name string, amount double);
+
+    @sink(type = 'log')
+    define stream LowProductionAlertStream (name string, amount double);
+
+     -- Simple Siddhi query to calculate production totals.
+     @info(name = 'query1')
+    from SweetsProductionStream
+    select name, sum(amount) as amount 
+	    group by name 
+    insert into ProductionAlertStream;
     ```
+    
+3. Save this file as `ManageProductionStats.siddhi`.
 
-    The above Siddhi application reads input data from a file named `production.csv` in the CSV format, processes it and publishes the resulting output in a Kafka topic named `total_production`. As a result, any application that cannot read streaming data, but is capable of subscribing to a Kafka topic can access the output. The each input event reports the amount of a specific sweet produced in a production run. The Streaming Integrator calculates the total produced of each sweet with each event. Therefore, each output event reports the total amount produced for a sweet from the time you started running the Siddhi application.
+4. Click on **Deploy** and then **Deploy To Server** to deploy the Siddhi Application in Streaming Integrator.
 
-3. Save this file as `ManageProductionStats.siddhi` in the `<SI_HOME>/wso2/server/deployment/siddhi-files` directory.
+4. Add Streaming Integrator server details by clicking on `Add New Server`
 
-    This deploys the `ManageProductionStats` in the Streaming Integrator. The following message appears to indicate that the Siddhi application is successfully installed.
+    Specify the Streaming Integrator host `localhost` and port `9443`.
+    
+5. Select the `ManageProductionStats` and the Server and **Deploy**.
```

Comment:
Is it Server URL or Server Name?

Reply:
it is a Server  we have added in the above step to select.

Reply:
Ok, let's specify that.